Header          hdr
int32[6]        dac_val
float32[6]      mpos
float32[6]      mvel

# float32[6]      jpos
# float32[6]      jvel
# float32[6]      j_enc_pos
